update-scripts/version/openssl

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+// The `openssl` version is not actually stored in the `pkgver` variable, but in
+// `_ver` (because of the funny trailing letter in the OpenSSL v1.1.* versions)
+
+(async () => {
+    const version = process.argv[2].replace(/_P/, 'p')
```

Comment:
Thos seems to be a leftover from OpenSSH.

Reply:
It totally is! Thank you for that.

Reply:
@rimrul this is now [fixed](https://github.com/git-for-windows/git-for-windows-automation/compare/6e5b5499098331e04fa87e1279a38cc04a956f9a..cb2053fe974cbce4ea00f640ab46d10376e7e0cb)!

Reply:
You where a bit overzealous. I meant just the `.replace()` you got rid of the whole line. We still need the `version` variable later, to write it to the `_ver` line

Reply:
Oh my. 🤦

Reply:
[Fixed again](https://github.com/git-for-windows/git-for-windows-automation/compare/cb2053fe974cbce4ea00f640ab46d10376e7e0cb..f7c15cb1a7d28fabeeae4060d867f65611c07135).